src/containers/Movie/Movie.js

```diff
@@ -1,21 +1,32 @@
 import React from 'react';
 import './Movie.css';
-// import { connect } from 'react-redux';
+import { connect } from 'react-redux';
 import star from '../../images/MovieTracker_star.svg';
 
-const Movie = ({ id, title, poster_path, release_date, vote_average, overview }) => {
+const Movie = ({ movie_id, title, poster_path, release_date, vote_average, overview, makeFavorite, currentUser }) => {
+
   return (
     <section className="grow">
       <div className="movie-img">
         <img className="poster-img" alt="Movie Poster" src = {`https://image.tmdb.org/t/p/original/${poster_path}`}/>
       </div>
-      <img src={star} alt="Logo" className="movie-star"/>
+      <img src={star} alt="Logo" className="movie-star" onClick={() => makeFavorite({
```

Comment:
I think we could also grab the movie id and filter it out of the movies array, not sure if one way is faster or not really. Excited to see this working.